### DISCUSSION TITLE
Branch renaming of "master" to "main"

### DISCUSSION THREAD

**Discussion by Standard8** [protz/thunderbird-conversations#1586] [General]
Github has now [announced support for renaming of branches](https://github.blog/changelog/2021-01-19-support-for-renaming-an-existing-branch/).

I'm planning on renaming "master" to "main" sometime this weekend (23/24 Jan).

In theory GitHub should tell you what to do to update your local repositories.

**Comment by Standard8** [comment]
A bit late, but working on this now.

**Reply by Standard8** [reply]
All done. You can rename the branch in forks by using the github UI:

https://github.com/github/renaming/#renaming-existing-branches

Then use these commands to rename in your local clones:

```
git branch -m master main
git fetch origin
git branch -u origin/main main
```

**Reply by protz** [reply]
thanks!
